node/src/components/chain_synchronizer/operations.rs

```diff
@@ -2202,9 +2239,7 @@ mod tests {
     #[test]
     fn validates_signatures_sufficiency_for_sync_to_genesis() {
         let consensus_verdict = Ok(());
-        assert!(are_signatures_sufficient_for_sync_to_genesis(
-            consensus_verdict
-        ));
+        assert!(are_signatures_sufficient_for_sync_to_genesis(consensus_verdict).is_ok());
```

Comment:
If you make the test return a `Result` you can use `?` here instead of `assert!`. Then a failure would always print the unexpected error.

Reply:
Cool, TIL that you can return `Result` from a unit test :+1: 

In this case, however, I'd like to keep the asserts explicit for clarity, because I expect either `ok` or `error` depending on the case.